<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWidget</class>
 <widget class="QWidget" name="ConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>226</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_themes">
     <property name="text">
      <string>Apply theme:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comboBox_themes">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_propertyEditor">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Edit theme</string>
       </property>
       <property name="text">
        <string>âœŽ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_save">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_onTop">
     <property name="text">
      <string>Always on top:</string>
     </property>
     <property name="buddy">
      <cstring>checkBox_onTop</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="checkBox_onTop">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_center">
     <property name="text">
      <string>Center on show:</string>
     </property>
     <property name="buddy">
      <cstring>checkBox_center</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="checkBox_center">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_followMouse">
     <property name="text">
      <string>Show on current screen:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="checkBox_followMouse">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_hideOnFocusOut">
     <property name="text">
      <string>Hide on focus out:</string>
     </property>
     <property name="buddy">
      <cstring>checkBox_hideOnFocusOut</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_hideOnClose">
     <property name="text">
      <string>Hide on close:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_clearOnHide">
     <property name="text">
      <string>Clear on hide:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="checkBox_hideOnFocusOut">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="checkBox_hideOnClose"/>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="checkBox_clearOnHide"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_systemShadow">
     <property name="text">
      <string>System shadow:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QCheckBox" name="checkBox_systemShadow">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox_center</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkBox_followMouse</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_center</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_followMouse</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
